.github/workflows/pkgci_regression_test.yml

```diff
@@ -48,12 +48,12 @@ jobs:
             rocm-chip: gfx942
             backend: rocm
             sku: mi300
-            runs-on: nodai-amdgpu-mi300-x86-64
+            runs-on: linux-mi300-1gpu-ossci-iree-org
           - name: amdgpu_rocm_mi308_gfx942
             rocm-chip: gfx942
             backend: rocm
             sku: mi308
-            runs-on: nodai-amdgpu-mi308-x86-64
+            runs-on: linux-mi300-1gpu-ossci-iree-org
```

Comment:
This should still be a mi308 machine. Are there mi308 machines in the ossci cluster?

Reply:
The machine I'm working on only has mi300.  I'm not sure if we have a mi308 in the fleet